src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -253,19 +250,8 @@ private PodSpec createPodSpec(String appId, AppDeploymentRequest request, Intege
 		ImagePullPolicy pullPolicy = deduceImagePullPolicy(properties, request);
 		container.setImagePullPolicy(pullPolicy.name());
 		// Add volumes and mounts
-		if (properties.getHostVolumeMounts() != null) {
-			container.setVolumeMounts(properties.getHostVolumeMounts().stream()
-					.map(hvm -> new VolumeMount(hvm.getContainerPath(), hvm.getName(), hvm.isReadOnly(), null))
-					.collect(Collectors.toList()));
-			podSpec.withVolumes(properties.getHostVolumeMounts().stream()
-					.map(mv -> {
-						Volume volume = new Volume();
-						HostPathVolumeSource hostPath = new HostPathVolumeSource();
-						hostPath.setPath(mv.getHostPath());
-						volume.setHostPath(hostPath);
-						volume.setName(mv.getName());
-						return volume;
-					}).collect(Collectors.toList()));
+		if (properties.getVolumeMounts() != null) {
```

Comment:
The thinking was that volumes would be something that a cluster admin would create and would not change all that often. Happy to support volumes defined as deployment properties as well.

However, not sure how we would parse the format of the value passed in the deployment property?
I.e. we essentially get a String value which we would have to parse and build the `Volume` model with (the `volumeMount` format is consistent, hence [this format](https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/pull/86/files#diff-ac742fc1f690acd37859b18a8bbf479bR204)). The different volume types differ slightly in their configuration (e.g. `hostPath` vs. [`cephfs`](https://github.com/kubernetes/kubernetes/blob/master/examples/volumes/cephfs/cephfs.yaml#L14) so not sure what format to use to support the various volume types?